sdk/resourcemanager/Azure.ResourceManager.Core/src/ArmOperation.cs

```diff
@@ -28,5 +39,28 @@ protected ArmOperation(TOperations syncValue)
         /// Gets the <see cref="OperationsBase"/> representing the result of the ArmOperation.
         /// </summary>
         protected TOperations SyncValue { get; }
+
+        /// <summary>
+        /// Waits for the completion of the long running operations.
+        /// </summary>
+        /// <param name="cancellationToken"> A token to allow the caller to cancel the call to the service. The default value is <see cref="CancellationToken.None" />. </param>
+        /// <returns> A response with the <see cref="ArmOperation{TOperations}"/> operation for this resource. </returns>
+        /// <remarks>
+        /// <see href="https://azure.github.io/azure-sdk/dotnet_introduction.html#dotnet-longrunning">Details on long running operation object.</see>
+        /// </remarks>
+        public ArmResponse<TOperations> WaitForCompletion(CancellationToken cancellationToken = default)
+        {
+            var pollingInterval = ArmOperationHelpers<TOperations>.DefaultPollingInterval;
```

Comment:
>DefaultPollingInterval [](start = 67, length = 22)

do we have an ADO item on polling interval based on Retry-After AND/OR a method param override? Currently it is hardcode to 1s.